nuxeo-ai-model/src/main/java/org/nuxeo/ai/bulk/DataSetUploadComputation.java

```diff
@@ -44,8 +48,12 @@
 
     private static final Log log = LogFactory.getLog(DataSetUploadComputation.class);
 
+    protected final Timer uploadTimer;
+
     public DataSetUploadComputation(String name) {
         super(name, 1, 0);
+        MetricRegistry registry = SharedMetricRegistries.getOrCreate(MetricsService.class.getName());
+        uploadTimer = registry.timer(MetricRegistry.name("nuxeo", "ai", "cloud", "upload"));
```

Comment:
If I remember well, metric stuff can be registered only once per instance. You probably need to remove them when computation shutdown + make it unique per computation (if several are running on same node)

Reply:
You are right.  Its not needed so I will take it out.